move/example/sources/its/token.move

```diff
@@ -0,0 +1,66 @@
+module example::token;
+
+use axelar_gateway::channel::{Self, Channel};
+use sui::coin::{Self, CoinMetadata, TreasuryCap};
+
+// -------
+// Structs
+// -------
+public struct Singleton has key {
```

Comment:
there is no need for a singleton here. See the template interchain token package for how this should look like. Coin metadata and treasury cap should be transferred to the deployer. Channel is held by the app's singleton.